lib/components/Select.js

```diff
@@ -11,6 +11,18 @@ const sizes = {
   large: "large",
 };
 
+// Refer: https://github.com/bigbinary/neeto-ui/issues/123
+const CustomInput = (props) => {
+  const { selectProps } = props;
+  return (
+    <components.Input
+      {...props}
+      data-testid={selectProps && selectProps["data-testid"]}
+      data-cy={selectProps && selectProps["data-cy"]}
+    ></components.Input>
+  );
+};
+
```

Comment:
Can you please give a brief on what was the cause of the issue and how this change will fix it?

Reply:
Issue:
- The onBlur callback was not triggered after selecting an option. So that the touched property in formik context would't get update.
- Initially the CustomInput component was defined inside the Parent component
- So when we click on the the select component and select an option, the selected value changes and this lead to re-render the Parent component
- This will lead re-creating the CustomInput and will cleanup the focus listener in it. So after choosing an option, if we click outside of the Select component, the onBlur callback will not be called.

Solution:
- Moved the definition  of  CustomInput component to outside of the Parent Component.
- This will avoid re-creating the CustomInput while Parent re-renders.
- So even if we select and option and clicked outside of the component, the onBlur will get triggered. 
- And the touched property in Formik context will also get updated.